examples/juniper/cptx-host/cptx-host.pb.txt

```diff
@@ -5,12 +5,13 @@ nodes: {
     model: "cptx"
     os: "evo"
     config: {
+      image: "cptx:test"
       config_path: "/home/evo/configdisk"
       config_file: "juniper.conf"
       file: "juniper.config"
       cert: {
             self_signed: {
-                cert_name: "ca-ipsec",
+                cert_name: "grpc-server-cert",
```

Comment:
why rename this?

Reply:
ca-ipsec was not the apt name for this certificate as this is used for grpc